.github/actions/setup-godot-cpp/action.yml

```diff
@@ -0,0 +1,59 @@
+name: Setup godot-cpp
+description: Setup build dependencies for godot-cpp.
+
+inputs:
+  platform:
+    required: true
+    description: Target platform.
+  em-version:
+    default: 3.1.62
+    description: Emscripten version.
+  windows-use-mingw:
+    description: 'Set to true to use MinGW, false otherwise'
+    required: true
+    default: 'false'
```

Comment:
Maybe we want to set this to true by default / invert to "windows-use-msvc"? godot-cpp-template used MinGW for windows builds, and I heard it referred to as preferred over MSVC.

Reply:
I think mingw tends to make more optimized builds, and MSVC tends to build faster? If I've got that right then it probably does make sense to prefer mingw if the builds may be used in releases?

Reply:
Alright, I changed it to an enum, defaulting to `mingw`. I think more verbose is more better in this case; `use-mingw=false` would be kinda confusing. If the build is succeeding now, it's ready from my side!